presto-hive/src/main/java/com/facebook/presto/hive/HiveStorageFormat.java

```diff
@@ -91,6 +93,11 @@
     TEXTFILE(LazySimpleSerDe.class.getName(),
             TextInputFormat.class.getName(),
             HiveIgnoreKeyTextOutputFormat.class.getName(),
+            new DataSize(8, Unit.MEGABYTE)),
```

Comment:
It's based on HiveIgnoreKeyTextOutputFormat which simply write binary on my understanding. We do not use the write functionality but the unit test requires a valid StorageFormat for the test file format.